docker-compose.yml

```diff
@@ -5,11 +5,14 @@ services:
   server:
     build:
       context: ./ui
+    image: ${EXPERIMENT}/server
```

Comment:
maybe we don't need this in the ui part.

Reply:
The server service mounts the experiment data folder.
Don't think we need to name the image after the experiment as the image does not contain any experiment specific things.

Reply:
Indeed. Removed EXPERIMENT from image names for ui and model generation. Replaced with ``evidence``.